jobs/config.json

```diff
@@ -415,6 +415,141 @@
       "UNKNOWN"
     ]
   }, 
+  "ci-kubernetes-e2e-gce-cos-beta-k8s-current": {
```

Comment:
Is this something trivial to change? Otherwise, it would make the auto job generation mechanism harder to implement.